test/e2e/receive_test.go

```diff
@@ -602,4 +605,702 @@ func TestReceive(t *testing.T) {
 			},
 		})
 	})
+
+	t.Run("single_active_series_limiting", func(t *testing.T) {
```

Comment:
This adds a lot of new tests to maintain. 

Adding  limit check to our previous test is probably overkill given eventuallity, but maybe? I would be also fine with just one most complex arch test instead of five.

Reply:
So I've removed all except the multitenant hashring limiting test! 🙂